stories/useFriction.stories.tsx

```diff
@@ -3,49 +3,94 @@ import { withKnobs, number } from '@storybook/addon-knobs';
 
 import { useFriction } from '../src';
 import './index.css';
+import './toggle.css';
+import './button.css';
 
 export default {
   title: 'Friction',
   decorators: [withKnobs],
 };
 
-export const FrictionOpacity: React.FC = () => {
+export const FrictionMultiple: React.FC = () => {
+  const [toggle, setToggle] = React.useState(true);
+
   const [props] = useFriction<HTMLDivElement>({
-    from: { opacity: 0 },
-    to: { opacity: 1 },
+    from: {
+      opacity: toggle ? 0 : 1,
+      transform: toggle
+        ? 'scale(0) rotate(0deg) translate(-50%, -50%)'
+        : 'scale(1) rotate(180deg) translate(-50%, -50%)',
+    },
+    to: {
+      opacity: toggle ? 1 : 0,
+      transform: toggle
+        ? 'scale(1) rotate(180deg) translate(-50%, -50%)'
+        : 'scale(0) rotate(0deg) translate(-50%, -50%)',
```

Comment:
Changing `from` / `to` values will cause the animation to re-run, which is exactly what we want. This allows users to hook these values up to, say, a toggle, or an effect that runs the animations on an interval by changing the `from` / `to` values.